spec/features/blackout_spec.rb

```diff
@@ -54,4 +82,14 @@ def fill_in_date(field:, date:)
     find(:xpath, "//input[@id='date_#{field}_alt']", visible: :all)
       .set(date)
   end
+
+  def check_days
```

Comment:
This method should live inside the `describe 'recurring blackout'` block since it's specific to that scenario.